test/prefer-math-trunc.js

```diff
@@ -12,15 +12,26 @@ const ruleTester = avaRuleTester(test, {
 
 const MESSAGE_ID_BITWISE_OR = 'bitwiseOr';
 const MESSAGE_ID_BITWISE_NOT = 'bitwiseNot';
+const MESSAGE_ID_BITWISE_SIGNED_RIGHT_SHIFT = 'bitwiseSignedRightShift';
+const MESSAGE_ID_BITWISE_LEFT_SHIFT = 'bitwiseLeftShift';
+const MESSAGE_ID_BITWISE_XOR = 'bitwiseXor';
+const MESSAGE_ID_BITWISE_AND = 'bitwiseAnd';
 
 const errorsBitwiseOr = [{messageId: MESSAGE_ID_BITWISE_OR}];
 const errorsBitwiseNot = [{messageId: MESSAGE_ID_BITWISE_NOT}];
+const errorsBitwiseSignedRightShift = [{messageId: MESSAGE_ID_BITWISE_SIGNED_RIGHT_SHIFT}];
+const errorsBitwiseLeftShift = [{messageId: MESSAGE_ID_BITWISE_LEFT_SHIFT}];
+const errorsBitwiseXor = [{messageId: MESSAGE_ID_BITWISE_XOR}];
+const errorsBitwiseAnd = [{messageId: MESSAGE_ID_BITWISE_AND}];
```

Comment:
The `invalid` cases for `ruleTester` can test with `errors: 1`, but make sure every message is tested in `visualizeTester`

Reply:
I'm not sure I understood, please tell me if my last 2 commits are not OK 😄

Reply:
Your commit is fine, but since we have `visualizeTester`, we can test messages in that, and in `ruleTester` you can use

```js
{
	code: 'foo',
	ouput: 'bar',
	errors: 3
}
```

To make it easier.

Reply:
I'm sorry but I'm not really familiar with the `visualizeTest`... How can I test the message inside? And why did you put `errors: 3` in the ruleTester? (I don't understand the `3`)

Reply:
number means numbers of errors

Reply:
> How can I test the message inside?

The snapshot is an assertion.